pages/dashboard/views/index.jsx

```diff
@@ -47,6 +48,29 @@ const EstablishmentPanel = ({ establishment, profile }) => {
   );
 };
 
+const getWarningText = review => {
+  let warningText = `${review.name} has `;
+
```

Comment:
```suggestion
  const plural = (item, count) `${item}${count === 1 ? '' : 's'}`;
```

And then scatter liberally below.

Reply:
we still need to handle `is/are` `it's/their` so it's probably simpler just to have separate sentences.

Reply:
Legit. Words are a pain.